tests/page-objects/forms/generic-form.po.js

```diff
@@ -1,15 +1,36 @@
+const { element } = require('protractor');
 const helper = require('../../helper');
 const utils = require('../../utils');
 const nameField = element(by.css('#report-form form [name="/data/name"]'));
 const submitButton = element(by.css('.enketo .submit'));
 const submittedName = element(by.css('#reports-content .details ul li:first-child p'));
+const formTitle = element(by.id('form-title'));
+const nextButton = element(by.css('button.btn.btn-primary.next-page'));
 
 const leftActionBarButtons = () => element.all(by.css('.general-actions .actions.dropup > a'));
 
+const fillForm = async (formFields) => {
+  for (const [fieldName] of Object.entries(formFields)) {
+    const field = formFields[fieldName];
```

Comment:
It is a little confusing to read this when it's destructuring to get `fieldName` from the "entry" and then it uses that to get the "value" anyway. At a minimum, you can do it all like this:

```suggestion
  for (const [fieldName, field] of Object.entries(formFields)) {
```